apps/docs/reference/presence.md

```diff
@@ -1,68 +1,150 @@
+	buttons: [
+		{ label: "Watch", url: "https://youtu.be/gQ3A9Ph1Nuo" },
+	],
+};
+
+// Update the user's activity
+presence.setActivity(presenceData);
+```
+
+::: details Example with Slideshow
+```ts
+// Creates a new slideshow
+const slideshow = presence.createSlideshow();
 
-### `new Presence(options: PresenceOptions)`
+// Defines two activities for the slideshow
+const likesPresence = { details: "Video has 1.2k likes!" };
+const viewsPresence = { details: "Video has 80k views!" };
 
-Creates a new `Presence` instance.
+// Add the two activities with 5 seconds intervals
+slideshow.addSlide("likeCounter", likesPresence, 5000);
+slideshow.addSlide("viewsCounter", viewsPresence, 5000);
 
-#### Parameters
+// Update the user's activity, toggling between the two every 5 seconds
+presence.setActivity(slideshow);
+```
+
+For more details check [Slideshow](#TODO)
+:::
 
-- `options` (`PresenceOptions`): The options for the Presence.
+## getStrings(strings: Record\<string,string\>)
 
-#### Returns
+Used to fetch translations from the extension.
 
-- `Presence`: The new `Presence` instance.
+a list of translations can be found in the [Translations Repository](#TODO)
 
-## Properties
+to add your own custom translations check [Custom Translations](#TODO)
 
-### `clientId: string`
 
-The Client ID of the Presence.
+```ts
+// type definition
+function getStrings<T extends Record<string, string>>(
+	strings: T,
+): Promise<T>;
+```
 
-## Methods
 
-### `setActivity(activity: PresenceActivity)`
+Example
+```ts {7-9}
+const strings = await presence.getStrings({
+	playing: "general.playing",
+	paused: "general.paused",
+	search: "general.search",
+}); // { playing: "Playing", paused: "Paused", search: "Searching" }
 
-Sets the activity of the Presence.
+console.log(strings.paused);    // EN: Paused
+console.log(strings.playing);   // DE: Spielt
+console.log(strings.search);    // RU: Ищет 
+```
 
-#### Parameters
+## clearActivity()
+
+Clears the user's activity, can be used when the user is idle (e,g: video paused)
+
+Example
+```ts{4-5}
+presence.on("UpdateData", () => {
+    const videoElement = document.querySelector("video");
+
+	// if the video is paused, clear the user's activity
+	if (videoElement.paused) return presence.clearActivity();
+
+    const presenceData: PresenceData = {
+        details: "Watching a video",
+    };
+
+    presence.setActivity(presenceData);
+});
+
+```
 
-- `activity` (`PresenceActivity`): The activity to set.
 
-### `clearActivity()`
 
-Clears the activity of the Presence.
 
-### `on(event: string, listener: Function)`
+## [`Presence Options`](#presence-options)
+Options that change the behavior of the presence
 
-Adds a listener to an event.
+### [`clientId`](#clientId)
 
-#### Parameters
+- **Type:** `string`
+- **Required** 
 
-- `event` (`string`): The event to listen to.
+This property is required to make your presence work, because it uses your application id to display its logo and assets.
```

Comment:
This isn't true, should be more smth like this (or smth else)
```suggestion
This property is required to make your presence work, because it uses your application id to identify its self to discord.
```